analytics.js

```diff
@@ -1035,19 +1035,20 @@ class Analytics {
           this.processResponse(200, res);
         }
       }
-      return;
-    }
-
-    let configUrl = getConfigUrl(writeKey);
-    if (options && options.configUrl) {
-      configUrl = getUserProvidedConfigUrl(options.configUrl, configUrl);
+      // return;
     }
 
-    try {
-      getJSONTrimmed(this, configUrl, writeKey, this.processResponse);
-    } catch (error) {
-      handleError(error);
-    }
+    // #### Commented this out since we don't care about the config
```

Comment:
what's in the configUrl?

Reply:
Just for clarity, the configUrl returns information about the source the sdk is using and which destinations it is connected to. However it doesn't seem to add any meta data to the event that we send.